test/integration/targets/aws_lambda/tasks/main.yml

```diff
@@ -0,0 +1,381 @@
+---
```

Comment:
Let's rename it after these changes are merged. In https://github.com/ansible/ansible/pull/21532 the module was renamed and then had to be reverted (and then a follow up PR was submitted to redo the rename).